docs/markdown/Reference-tables.md

```diff
@@ -167,6 +171,8 @@ These are the parameter names for passing language specific arguments to your bu
 | Rust          | rust_args     | rust_link_args    |
 | Vala          | vala_args     | vala_link_args    |
 | Cython        | cython_args   | cython_link_args  |
+| NASM          | nasm_args     |                   |
+| MASM          | masm_args     |                   |
```

Comment:
Maybe explicitly say `N/A` for `linker name` column?

Reply:
done